docs/index.html

```diff
@@ -110,6 +110,8 @@ <h3 id="supported-locales">Supported Locales</h3>
 <li>tr - Turkish</li>
 <li>uk - Ukrainian</li>
 <li>zh_CN - Chinese (Simplified)</li>
+<li>cs - Czech</li>
```

Comment:
This file is automatically generated during build, from the README so these changes aren't necessary but it's fine as is because it will just be overwritten.